src/ripple/app/main/LoadManager.cpp

```diff
@@ -139,7 +139,7 @@ void LoadManager::run ()
                 // as undefined behavior.
                 //
                 constexpr auto deadlockTimeLimit = 90s;
-                assert (timeSpentDeadlocked < deadlockTimeLimits);
+                assert (timeSpentDeadlocked < deadlockTimeLimit);
```

Comment:
Nitpick:  To be consistent with the comment, and with `timeSpentDeadlocked > deadlockTimeLimit`, this should be `timeSpentDeadlocked <= deadlockTimeLimit`.

Reply:
I did the other condition, but same effect. I do like symmetry :)